tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -882,14 +882,22 @@ def _select_case(self, role, case):
         finder = re.compile(r'.*\b' + case + r'\b')
         finder_dotted = re.compile(r'.*\b' + case.replace(' ', r'\.') + r'\b')
         for elem in elems:
-            action_chains = ActionChains(self._browser)
-            action_chains.move_to_element(elem)
-            action_chains.perform()
-            logger.debug(elem.text)
-            if finder.match(elem.text) or finder_dotted.match(elem.text):
-                parent = elem.find_element_by_xpath('..')
-                checkbox = parent.find_element_by_class_name('tree-checkbox')
-                break
+            # perform() will throw exceptions if elem.text is null since Chrome
```

Comment:
```suggestion
            # elem.txt might be null when the required reference devices could
            # not be met (either due to quantity or the type) for specific test.
            # perform() will throw exceptions if elem.text is null since Chrome
```